.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+Fixes # [insert issue number].
```

Comment:
I'd probably move this to the bottom of the template

Reply:
👍 done :)